launcher/src/main/java/org/apache/spark/launcher/package-info.java

```diff
@@ -21,12 +21,14 @@
  * <p>
  * This library allows applications to launch Spark programmatically. There's only one entry
  * point to the library - the {@link org.apache.spark.launcher.SparkLauncher} class.
+ * Under YARN manager cluster mode, it supports launching in Application in thread or
```

Comment:
Delete this sentence. It's actually not correct on top of being a little confusing.

You can launch any application in a child thread. What YARN cluster mode currently gives you is that it's safe to launch multiple applications as child threads in the same process.